src/content/en/2021/caching.md

```diff
@@ -17,4 +20,470 @@ featured_stat_3: TODO
+
+When both are specified, `ETag` takes precedence.
+
+{{ figure_markup(
+  image="last-modified-etag.png",
+  caption="Percent of responses that set `Last-Modified` and `ETag` headers.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1490687174&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+Comparing 2020 and 2021, we notice a recurring trend from past years with the `ETag` becoming slightly more popular, and `Last-Modified` being used 1.5% less. What we should probably keep an eye on next year is a new trend of 1.4% more pages using neither `ETag` nor `Last-Modified` headers, as this could imply a challenge in the community understanding the value of these headers.
+
+### `Cache-Control` directives
+When using the `Cache-Control` header, you specify one or more *directives*—predefined values that indicate specific caching functionality. Multiple directives are separated by commas and can be set in any order, although some clash with one another (e.g., `public` and `private`). In addition, some directives take a value, such as `max-age`.
+
+Below is a table showing the most common `Cache-Control` directives:
+
+<table>
+  <tr>
+   <th>Directive</th>
+   <th>Description</th>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">max-age</code></td>
+   <td>Indicates the number of seconds that a resource can be cached for relative to the current time. For example <code>max-age=86400</code>.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">public</code></td>
+   <td>Indicates that any cache can store the response, including the Browser and the CDN. This is assumed by default.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">no-cache</code></td>
+   <td>A cached resource must be revalidated before its use, via a conditional request, even if it is not marked as stale.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">must-revalidate</code></td>
+   <td>A stale cached entry must be revalidated before its use, via a conditional request.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">no-store</code></td>
+   <td>Indicates that the response must not be cached.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">private</code></td>
+   <td>The response is intended for a specific user and should not be stored by shared caches such as CDNs.</td>
+  </tr>
+  <tr>
+   <td><code class="no-wrap">immutable</code></td>
+   <td>Indicates that the cached entry will never change during its TTL and that revalidation is not necessary.</td>
+  </tr>
+</table>
+
+The `max-age` directive is the most commonly found with 62.2% of Desktop requests include a `Cache-Control` response header with this directive.
```

Comment:
Edit: 

The `max-age` directive is the most commonly found with 62.2% of Desktop requests including a `Cache-Control` response header with this directive.